build.gradle

```diff
@@ -87,99 +87,103 @@ ext {
     generatedMetamodels = "${buildDir}/generated/api/src/gen/java"
 }
 
-dependencies {
-    //Specify BOMs that are used for specifying versions of libraries
-    implementation platform('org.jboss.resteasy:resteasy-bom:4.4.3.Final')
-    implementation platform('com.fasterxml.jackson:jackson-bom:2.12.3')
-    implementation platform('com.google.inject:guice-bom:4.2.3')
-    implementation platform('org.junit:junit-bom:5.7.0')
+ext {
+    // Denotes BOM versions
+    resteasy_version = "4.4.3.Final"
+    jackson_version = "2.12.3"
+    guice_version = "4.2.3"
+    junit_version = "5.7.0"
+}
 
+dependencies {
     // Commons
     implementation "commons-codec:commons-codec:1.11"
     implementation "commons-collections:commons-collections:3.2.2"
     implementation "commons-io:commons-io:2.7"
     implementation "commons-lang:commons-lang:2.5"
 
-    //collections
+    // Collections
     implementation "com.google.guava:guava:30.1-jre"
 
     // Gettext libraries used for internationalization & translation
     implementation "com.googlecode.gettext-commons:gettext-commons:0.9.8"
 
-    //Guice Libraries - Version from BOM
-    implementation "com.google.inject.extensions:guice-assistedinject"
-    implementation "com.google.inject.extensions:guice-servlet"
-    implementation "com.google.inject.extensions:guice-throwingproviders"
-    implementation "com.google.inject.extensions:guice-persist"
-
-    //Jackson - Version from BOM
-    implementation "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider"
-    implementation "com.fasterxml.jackson.module:jackson-module-jsonSchema"
-    constraints {
-        implementation "javax.validation:validation-api:2.0.1.Final"
-    }
+    // Guice Libraries
+    implementation "com.google.inject.extensions:guice-assistedinject:$guice_version"
+    implementation "com.google.inject.extensions:guice-servlet:$guice_version"
+    implementation "com.google.inject.extensions:guice-throwingproviders:$guice_version"
+    implementation "com.google.inject.extensions:guice-persist:$guice_version"
 
-    implementation "com.fasterxml.jackson.datatype:jackson-datatype-hibernate5"
-    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jdk8"
-    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
-    implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml"
-    implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-xml"
+    // Jackson
+    implementation "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:$jackson_version"
+    implementation "com.fasterxml.jackson.module:jackson-module-jsonSchema:$jackson_version"
 
-    //Javax
+    // Bean validation API is explicitly added to this version,
```

Comment:
Please call out that this is a transitive dependency of com.fasterxml.jackson.module:jackson-module-jsonSchema in the comment.